README.md

```diff
@@ -16,16 +16,25 @@ The FastAPI application provides a simple endpoint at the root (`/`) that return
 
 1. Open the project in VSCode.
 2. When prompted, open the project inside the container. (Or press `F1`, type `Remote-Containers: Reopen in Container`).
-3. Once inside the container, VSCode will automatically bind the project directory and forward port `5001` for the FastAPI application.
-4. Visit [Localhost:5001](http://localhost:5001/) in your browser to see the current Unix timestamp returned by the FastAPI app.
-5. Access the API Swagger docs [here](http://localhost:5001/docs).
+3. Once inside the container, open VSCode Terminal and run these commands:
+```
+cd /workspaces/fastapi-example/src
```

Comment:
don't use absolute paths here, they only make sense within a devcontainer.

you can assume the user is in the root of the project, so :

```
cd src
```

Reply:
done